docs/pages/api-reference/liveblocks-yjs.mdx

```diff
@@ -151,7 +184,7 @@ fire events when the document has loaded.
 
 ```ts
 // Listen for the sync event only once
-yProvider.on("sync", (isSynced: boolean) => {
+yProvider.once("sync", (isSynced: boolean) => {
```

Comment:
Not a comment, just something that crossed my mind - we don't have proper Typescript autocompletion for these event subscription :(
I hope we can improve that soon in the future